src/publish/confluence/confluence-helper.ts

```diff
@@ -614,8 +622,10 @@ export const updateImagePaths = (body: ContentBody): ContentBody => {
 export const findAttachments = (
   bodyValue: string,
   publishFiles: string[] = [],
-  filePath: string = ""
+  filePathParam: string = ""
 ): string[] => {
+  const filePath = filePathParam.replaceAll("\\", "/");
```

Comment:
I tried Deno's path attributes and ran into some issues so I'm now adding tests and doing string replacement.  I'm not super happy with this, I want it to be all Deno.